node/processingOnlyNode/statusComponents.go

```diff
@@ -0,0 +1,53 @@
+package processingOnlyNode
+
+import (
+	"time"
+
+	outportCfg "github.com/multiversx/mx-chain-core-go/data/outport"
+	"github.com/multiversx/mx-chain-go/common"
+	"github.com/multiversx/mx-chain-go/common/statistics"
+	"github.com/multiversx/mx-chain-go/factory"
+	"github.com/multiversx/mx-chain-go/integrationTests/mock"
+	"github.com/multiversx/mx-chain-go/outport"
+	"github.com/multiversx/mx-chain-go/testscommon"
+)
+
+type statusComponentsHolder struct {
+	outportHandler         outport.OutportHandler
+	softwareVersionChecker statistics.SoftwareVersionChecker
+	managedPeerMonitor     common.ManagedPeersMonitor
+}
+
+func CreateStatusComponentsHolder(shardID uint32) (factory.StatusComponentsHolder, error) {
```

Comment:
missing comments on all exported items in this file

Reply:
done